include/libcyphal/transport/can/media.hpp

```diff
@@ -33,13 +33,17 @@ struct RxMetadata final
     std::size_t payload_size;
 };
 
+/// @brief Defines interface to a custom CAN bus media implementation.
+///
+/// Implementation is supposed to be provided by an user of the library.
```

Comment:
probably, f.e. for SocketCAN media

Reply:
@lydia-at-amazon I think it could fit as part of stage F "Design review"

Reply:
I think that's appropriate. For selfish reasons though, I would be interested in trying to get a concrete example running a little sooner to uncover any problems and to gauge how user-friendly implementing the Media and Multiplexer classes are. It would also help us make sure there are no gaps in our API. We could talk about this in the dev call. (I might not be able to make this Friday's dev call so maybe the next next dev call).